roles/horizon/tasks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: set up log rotation for horizon
+  logrotate: name=horizon path=/var/log/apache2/*.log
+  args:
+    options:
+      - daily
+      - missingok
+      - rotate 7
+      - compress
```

Comment:
i think easiest solution is i can do it by just putting the custom logrotate conf in a file in roles/horizon/templates, and installing it that way, because it seems this file gets set when we install apache2. but im not sure that's exactly something we'd want to do, what are your thoughts @j2sol